src/hyperswarm-dht-logger.js

```diff
@@ -0,0 +1,79 @@
+'use strict'
+
+const debug = require('debug')('hcore-logger')
+const DHT = require('@hyperswarm/dht')
+const Tail = require('tail').Tail
+const { crypto_sign_SEEDBYTES } = require('sodium-universal') // eslint-disable-line
+
+class HyperSwarmDHTLogger {
+  /**
+   * @param {string} file
+   * @param {string|null} seed
+   */
+  constructor (file, seed = null) {
+    this.file = file
+    this.seed = seed && Buffer.allocUnsafe(crypto_sign_SEEDBYTES).fill(seed)
```

Comment:
I would suggest just to use 32 instead of importing a lib for this

Reply:
do you think it's reasonable to use magic number in this case?

Reply:
well that seems to be the number of constant and instead of importing a full library for it we could simply declare the const here (`const CRYPTO_SIGN_SEEDBYTES = 3`)

Reply:
>  importing a full library

it is already imported by @hyperswarm/dht

Reply:
afaik not but my main issue is that we'll have to maintain dependencies on two pacakges then (`@hyperswarm/dht` and current package), and if npm audit sees an issue in package then we have to make updates and releases on both

Reply:
ah.. I forgot to add `sodium-universal` as a direct dependency, never mind